examples/object_detection_2d/object_detection_2d/seed_test_run.py

```diff
@@ -121,7 +120,6 @@ def setup_evaluator() -> ObjectDetectionEvaluator:
                 min_confidence_score=0.2,
             ),
             ThresholdConfiguration(
-                threshold_strategy=F1_OPTIMAL,
```

Comment:
For the example, wouldn't it be better to specify the `threshold_strategy` just to make it explicit?